src/modules/statement/stmt.rs

```diff
@@ -71,8 +70,7 @@ pub enum StatementType {
     Exit(Exit),
     CommandModifier(CommandModifier),
     Comment(Comment),
-    CommentDoc(CommentDoc),
-    ConstInit(ConstInit),
+    CommentDoc(CommentDoc)
```

Comment:
Please reinstate the trailing comma; it makes PR diffs slightly nicer, and makes keystroke macros easier to run without having to account for the last item.